packages/services/src/terminal/terminal.ts

```diff
@@ -145,23 +145,21 @@ export interface IManager extends IBaseManager {
   /**
    * Create a new terminal session.
    *
-   * @param options - The options used to create the session.
+   * @param name - The name of the target terminal.
    *
-   * @returns A promise that resolves with the terminal instance.
+   * @returns A promise that resolves with the terminal connection instance.
    *
    * #### Notes
    * The manager `serverSettings` will be always be used.
    */
-  startNew(
-    options?: ITerminalConnection.IOptions
-  ): Promise<ITerminalConnection>;
+  startNew(name?: string): Promise<ITerminalConnection>;
```

Comment:
The options are indeed not used in the implementation. But I would recommend still passing an option object (with a new interface to define). To be coherent with other APIs in JupyterLab. Something like:

```ts
startNew(options?: ITerminal.IOptions): Promise<ITerminalConnection>;

export namespace ITerminal {
  export interface IOptions {
    name?: string;
  }
}
```

Reply:
In https://github.com/jupyterlab/jupyterlab/pull/11637/commits/3e83c95e63b22b59191075bb22d3abe5e413f110, I used the existing `ITerminalConnection.IOptions` as it's done for `connectTo`. What do you think?

Reply:
I'm not a fan because it make the API unclear. You are passing a fake model just to get a name. But from the 3e83c95 API one would wonder, why do I need a model to start a new terminal? Should I first instantiate some model?

Reply:
I agree, but we could also say the same for `connectTo` (we create a "fake" model just to pass a name). Do you think we should also refactor the `connectTo` API?

Reply:
No

Conceptually, sessions and terminals are similar. You start a new one that generates a model. And you can use that model to connect to the corresponding server resources.

We should stick with passing the model in `connectTo` even if the model is really simple for terminals.

Reply:
Thanks, then https://github.com/jupyterlab/jupyterlab/pull/11637/commits/f4bd0ec62ad6903f381bc26bd6d75f1b7d1cca92 should be enough.